server/src/main/java/io/spine/server/commandbus/CommandDispatchingException.java

```diff
@@ -20,15 +20,19 @@
 
 package io.spine.server.commandbus;
 
-import org.junit.jupiter.api.DisplayName;
-
 /**
- * @author Alexander Yevsyukov
+ * A {@link RuntimeException} thrown when a {@link CommandDispatcher} fails to dispatch a command.
+ *
+ * <p>This exception <b>must</b> have a {@linkplain #getCause() cause}, which explains why
+ * the dispatching was failed.
```

Comment:
Maybe `the dispatching failed`?

"The dispatching fails" (by itself) vs "the dispatching is failed" (by someone).